test/functional/test.js

```diff
@@ -0,0 +1,63 @@
+import { Selector, t } from 'testcafe';
+import fs from 'fs';
+import Server from '../../src';
+import os from 'os';
+import path from 'path';
+import assert from 'assert';
+
+let server = null;
+let files  = [];
+
+
+async function waitForFileDownload (pathFile) {
+    for (let i = 0; i < 10; i++) {
+        if (fs.existsSync(pathFile))
+            return true;
+        await t.wait(500);
+    }
+    return fs.existsSync(pathFile);
+}
+
+fixture`Download and Upload`
+    .page`http://localhost:1337`
+    .before(async () => {
+        server = new Server();
+
+        server.start();
+    })
+    .after(async () => {
+        for (let i = 0; i < files.length; i++)
+            fs.unlinkSync(files[i]);
```

Comment:
we can use `forEach` (it's just shorter) because we use sync `unlinkSync` function